### DISCUSSION TITLE
Não consigo dar push na minha branch

### DISCUSSION THREAD

**Discussion by Akira8938** [digitalinnovationone/dio-lab-open-source#51191] [Desafios DIO]
Não estou conseguindo dar push na minha branch, já verifiquei minhas credenciais e estão ok! Alguém pode me ajudar?

![image](https://github.com/user-attachments/assets/d09b6523-f50b-47f2-b214-84b89ba4fb90)

**Comment by gustavolima007** [comment]
Parece que teve um problema na autenticação via navegador.

Tente forçar a autenticação manual:

`git config --global credential.helper manager
git push origin feat/community/Akira8938`
